assets/_sass/custom-viz/boeing/boeing.scss

```diff
@@ -0,0 +1,66 @@
+/*==============================
+=          Animations          =
+==============================*/
+
+@keyframes wobble {
```

Comment:
These are just test animations, not necessarily to be used in the final piece. I was just testing css animations!